clientSide/src/components/IndividualBooking/CarRecord.jsx

```diff
@@ -0,0 +1,40 @@
+import { Link } from "react-router-dom";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faEdit } from "@fortawesome/free-solid-svg-icons";
+
+const CarRecord = ({ booking, bookingId, car }) => {
+  return (
+    <table>
+      <thead>
+        <tr>
+          <th>Model </th>
+          <th>Color</th>
```

Comment:
### TODO:
We need to extract the thead and make it a component since it is also reused in the cartable.jsx.
Moreover, we should at a parking slot to the cartable.jsx file.